apps/charterafrica/src/pages/api/v1/resources/ecosystem/[action].page.js

```diff
@@ -1,10 +1,18 @@
 import { schema } from "@/charterafrica/lib/ecosystem/airtable";
+import {
+  updateEcosystemList,
+  updateEcosystemContent,
+} from "@/charterafrica/lib/ecosystem/ecosystem";
 
 const isApiKeyValid = (key) => {
   return key && key === process.env.RESOURCES_SECRET_TOKEN;
 };
 
-const actionMap = { schema };
+const actionMap = {
+  "update-ecosystem-list": updateEcosystemList,
+  "update-ecosystem-content": updateEcosystemContent,
```

Comment:
```suggestion
  "update-list": updateEcosystemList,
  "update-content": updateEcosystemContent,
```
The endpoint is already `/ecosystem`, hence `/ecosystem/update-list` should be enough.